### ISSUE TITLE
Feature Request: Implement API calls and spacecmd commands for Virtual Host Managers

### ISSUE THREAD

**Issue by stdevel** [uyuni-project/uyuni#1258]
Hello Uyuni team!

It would be great to have API calls for managing Virtual Host Managers. Currently, there are no API calls or ``spacecmd`` commands for this purpose requiring manual action.

Best wishes,
Christian.

**Comment by mcalmer** [comment]
Check 

rhn/apidoc/handlers/VirtualHostManagerHandler.jsp

spacecmd support does not exist yet,

**Comment by stdevel** [comment]
Good to know - thanks for sharing! 👍 

Is there a dedicated repository for ``spacecmd`` along with Uyuni? I know that the original source code came from Spacewalk project so they surely won't implement this feature?

**Comment by mcalmer** [comment]
Yes, here in our git repository. The subdirectory `spacecmd/` contains this code and now also Unit tests.  Contribution is welcome.
